test/mocks/router/mocks.cc

```diff
@@ -16,7 +16,6 @@ namespace Envoy {
 namespace Router {
 
 MockDirectResponseEntry::MockDirectResponseEntry() = default;
-MockDirectResponseEntry::~MockDirectResponseEntry() = default;
```

Comment:
I believe the destructors are defined in the .cc to improve compilation performance.

Reply:
reverted. I think inlining almost empty ctors/de-ctors may even speed up compilation as it's less call-tos, but I have no concrete data. :)